tools/BuildAssets/psModules/CodeGenerationModules/generateDotNetSdkCode.psm1

```diff
@@ -0,0 +1,207 @@
+$errorStream = New-Object -TypeName "System.Text.StringBuilder";
+$outputStream = New-Object -TypeName "System.Text.StringBuilder";
+
+
+function Clear-OutputStreams {
+    $errorStream.Clear() | Out-Null
+    $outputStream.Clear() | Out-Null
+}
+
+function Write-InfoLog {
+    param(
+        [string] $msg,
+        [switch] $logToConsole,
```

Comment:
@dsgouda ideally one switch parameter would have been sufficient here.
you always want to log to console, so having -logToFile would be enough.
Something to consider when you are thinking of modify this module in the future.